src/System.Net.Http/src/uap/System/Net/HttpClientHandler.cs

```diff
@@ -701,6 +657,15 @@ private void SetOperationStarted()
                     _rtFilter.IgnorableServerCertificateErrors.Add(RTChainValidationResult.Untrusted);
                     _rtFilter.IgnorableServerCertificateErrors.Add(RTChainValidationResult.WrongUsage);
                     _rtFilter.ServerCustomValidationRequested += RTServerCertificateCallback;
+
+                    _rtFilterWithNoCredentials.IgnorableServerCertificateErrors.Add(RTChainValidationResult.Expired);
+                    _rtFilterWithNoCredentials.IgnorableServerCertificateErrors.Add(RTChainValidationResult.IncompleteChain);
+                    _rtFilterWithNoCredentials.IgnorableServerCertificateErrors.Add(RTChainValidationResult.InvalidName);
+                    _rtFilterWithNoCredentials.IgnorableServerCertificateErrors.Add(RTChainValidationResult.RevocationFailure);
+                    _rtFilterWithNoCredentials.IgnorableServerCertificateErrors.Add(RTChainValidationResult.RevocationInformationMissing);
+                    _rtFilterWithNoCredentials.IgnorableServerCertificateErrors.Add(RTChainValidationResult.Untrusted);
+                    _rtFilterWithNoCredentials.IgnorableServerCertificateErrors.Add(RTChainValidationResult.WrongUsage);
+                    _rtFilterWithNoCredentials.ServerCustomValidationRequested += RTServerCertificateCallback;
```

Comment:
If for some reason both filters are active, this would cause the callback to be called several times causing race conditions that could corrupt the application state.